data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,102 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Hai Pilgrimage"
+	name "Hai spiritual leader to <planet>"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	clearance "You inform the authorities of the passenger you are transporting, and they quickly allow you to land."
+	source
+		government "Hai"
+		not attributes "station"
```

Comment:
The only thing I have left to say on this is that given the nature of the setting (an open air port and a climate capable of producing thick fog), it would be worth reading through the planet descriptions and either creating a list of possible offer locations, or at the very least excluding the most egregiously incompatible ones (i.e. Stonebreak, Darkmetal, etc.).